src/Symfony/Component/Translation/Translator.php

```diff
@@ -399,51 +395,7 @@ private function initializeCacheCatalogue($locale, $forceRefresh = false)
         }
 
         $this->catalogues[$locale] = $catalogue;
-    }
-
-    private function getFallbackContent(MessageCatalogue $catalogue)
-    {
-        if (!$this->debug) {
-            // merge all fallback catalogues messages into $catalogue
```

Comment:
this part is important can your revert it ? see https://github.com/symfony/symfony/pull/13981